recipes/usgscsm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "usgscsm" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3a9617288cf70a0d3752a1e81eec9adacabf59092f2069e8635f5d91ce77d9d1
+
+build:
+  number: 0 
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [linux]
+    - cmake >=3.10
+  host:
+    - csm
+    - vc 14  # [win]
```

Comment:
Instead of doing this, please skip it for ``vc`` different of ``14``, such as:
```
skip: True  # [vc!=14]
```
or you may sip just for the version less than 14
```
skip: True  # [vc<14]
```